sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1229,7 +1229,7 @@ class Analyzer(override val catalogManager: CatalogManager)
         val partCols = partitionColumnNames(r.table)
         validatePartitionSpec(partCols, i.partitionSpec)
 
-        val staticPartitions = i.partitionSpec.filter(_._2.isDefined).mapValues(_.get).toMap
+        val staticPartitions = i.partitionSpec.filter(_._2.isDefined).mapValues(_.get)
```

Comment:
I remember this was for Scala 2.13 compliance. Does it work with Scala 2.13?

Reply:
i will close it.